app/test/integration_test.go

```diff
@@ -53,7 +53,7 @@ func NewIntegrationTestSuite(cfg cosmosnet.Config) *IntegrationTestSuite {
 
 func (s *IntegrationTestSuite) SetupSuite() {
 	if testing.Short() {
-		s.T().Skip("skipping block size integration test in short mode.")
+		s.T().Skip("skipping app/test/integration_test in short mode.")
```

Comment:
[optional nit]
this feels out of scope of this PR, also what is the ".../integration_test" supposed to refer to? isn't the file named something else?

Reply:
It was named block_size_test or similar, but since I added the proofs test there too, I changed it to integration_test